templates/oauth2_provider/authorize.html

```diff
@@ -0,0 +1,38 @@
+{% extends "oauth2_provider/base.html" %}
+{% load i18n %}
+
+{% block content %}
+<div class="block-center">
+    <form id="authorizationForm" method="post">
+        <h3>{% translate "Authorize" %} {{ application.name }}?</h3>
+        {% csrf_token %}
+
+        {% for field in form %}
+            {% if field.is_hidden and field.name != 'teams' %}
+                {{ field }}
+            {% endif %}
+        {% endfor %}
```

Comment:
What's the deal with this?

Also, any reason not to use our `render_form_fields` or `render_field` tags?